src/js/outreach/apps/verify_app.js

```diff
@@ -21,6 +21,8 @@ export default App.extend({
     return getPatientInfo({ actionId });
   },
   onFail(options, response) {
+    if (response.status >= 500) return;
```

Comment:
Does it even get here?  I would kind of assume the route takes over first, but I guess because there's no an error router app, it isn't stopping the previous app.. hmm that's interesting..